### DISCUSSION TITLE
Resolve server by hostname from Android

### DISCUSSION THREAD

**Discussion by lgordon** [jellyfin/jellyfin-desktop#507] [General]
I have the android client on an Nvidia shield and I can't reach my server by hostname, from a web browser on my laptop I can resolve the server by hostname perfectly. Is there some way I need to set this up? It works by IP but my router is limited to dynamic IPs which change so its annoying to keep the jellyfin client connected.

**Comment by iwalton3** [comment]
Please note this discussion board is only for discussing Jellyfin Media Player problems. (Actual bugs should be posted to the [Issues](https://github.com/jellyfin/jellyfin-media-player/issues) section.)

For help with Jellyfin, please see:
 - [Jellyfin Forums](https://forum.jellyfin.org/)
 - [Jellyfin Matrix Chat](https://jellyfin.org/contact)

**Comment by oddstr13** [comment]
Depending on Android version, your device might not support looking up mDNS hostnames (.local), and you would be dependent on the router providing a hostname for you.

I would suggest you look into setting a static IP on the Jellyfin server, you should be able to do this regardless of your routers capabilities.
Home routers usually give out dynamic IPs in a range similar to 192.168.1.100-192.168.1.250, with a hostmask of 255.255.255.0 (a /24 subnet), the IP addresses between 192.168.1.2-192.168.1.99 tend to be available for setting static IPs, but the exact setup varies from router to router, and from ISP to ISP. (the router's IP would be 192.168.1.1 in this example)  I'd suggest an IP around 192.168.1.20 if that's in your subnet range, and not available via DHCP from your router.
Remember to set the subnet mask, default route (your router) and DNS servers when setting static IPs.

I also advise having an additional computer available for searching the internet, should you screw up the configuration and need more information for fixing it (usually as easy as turning DHCP/dynamic config back on though).


In any case, your problem isn't with the Jellyfin server, but a generic networking question, so this might not be the best place to get help.
